src/Compilers/CSharp/Portable/Declarations/DeclarationTreeBuilder.cs

```diff
@@ -987,7 +987,7 @@ private static bool CheckMemberForAttributes(Syntax.InternalSyntax.CSharpSyntaxN
                     int numFieldDeclarators = fieldDeclarators.Count;
                     for (int i = 0; i < numFieldDeclarators; i++)
                     {
-                        set.TryAdd(fieldDeclarators[i].Identifier.ValueText);
```

Comment:
💡 `TryAdd` may be an extension method for `ImmutableSegmentedDictionary<TKey, VoidResult>.Builder`. If it's no longer needed, perhaps it can be removed?

Reply:
Done.